src/styles/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: var(--basic-margin-top) auto 0;
+  display: block;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 40px;
+  color: var(--first-color);
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--second-color);
+}
+
+.card__reviews {
+  margin-top: var(--basic-margin-top);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__reviews-paragraph {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--second-color);
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__price-paragraph {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--second-color);
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--first-color) ;
+}
```

Comment:
maybe we can add color to html, and change it only where we need it

Reply:
It is more convenient to change its value from a file with variables